imagetest/test_suites/network_perf/startupscripts/netserver_startup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# This script installs iperf on a VM and starts an iperf server for the client
+# to test the network bandwidth between the two VMs.
+
+sleepduration=120
+
+if [[ -f /usr/bin/apt ]]; then
+  echo "apt found Installing iperf."
+  sudo apt update && sudo apt install iperf
+elif [[ -f /bin/dnf ]]; then
+  echo "dnf found Installing iperf."
+  os=$(cat /etc/redhat-release)
+  if [[ "$os" == *"release 9."* ]]; then
+    sudo dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+  fi
+  if [[ "$os" == *"release 8."* ]]; then
+    sudo dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+  fi
+  sudo sudo dnf makecache && sudo dnf -y install iperf
+elif [[ -f /bin/yum ]]; then
+  echo "yum found Installing iperf."
+  yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  sudo sudo yum makecache && sudo yum -y install iperf
+elif [[ -f /usr/bin/zypper ]]; then
+  echo "zypper found Installing iperf."
+  sudo zypper --no-gpg-checks refresh
+  sudo zypper --no-gpg-checks --non-interactive install iperf
```

Comment:
For SLES, I found we can use the PackageHub repo which is version and arch dependent.

For SLES 15 SP5 we have to do the following.

```
SUSEConnect --product PackageHub/15.5/x86_64
zypper install iperf
```

For SLES 12 SP5
```
SUSEConnect --product PackageHub/12.5/x86_64
zypper install iperf
```

Note that those versions and architectures will have to be handled.

Reply:
Done.